From dcc854fd494b9127e1ce4f3a272cc315ca00de6d Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 16 Sep 2017 15:20:04 +0300
Subject: [PATCH 127/249] lineage: Remove libphonenumbergoogle

* It was only used by TextSecure, which is long deprecated

Change-Id: I334cb25c240351b47f9337b99c1e8e3f53401af9
---
 snippets/lineage.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index ce67068..123e883 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -13,7 +13,6 @@
   <!--<project path="external/koush/ion" name="LineageOS/ion" />-->
   <project path="external/libncurses" name="LineageOS/android_external_libncurses" />
   <project path="external/libnfc-nxp" name="LineageOS/android_external_libnfc-nxp" />
-  <!--<project path="external/libphonenumbergoogle" name="LineageOS/android_external_libphonenumbergoogle" />-->
   <project path="external/libtar" name="LineageOS/android_external_libtar" />
   <!--<project path="external/libtruezip" name="LineageOS/android_external_libtruezip" />
   <project path="external/lzo" name="LineageOS/android_external_lzo" />-->
-- 
2.17.1

